editor/animation_track_editor.cpp

```diff
@@ -4895,6 +4905,15 @@ float AnimationTrackEditor::snap_time(float p_value) {
 	return p_value;
 }
 
+void AnimationTrackEditor::_show_imported_anim_warning() const {
+
+	EditorNode::get_singleton()->show_warning(TTR("This animation belongs to an imported scene, so changes to existing tracks will not be saved.\n\n"
+												  "To enable the ability to add custom tracks, navigate to the scene's import settings and set\n"
+												  "\"Animation > Storage\" to \"Files\", enable \"Animation > Keep Custom Tracks\", then re-import.\n"
+												  "Alternatively, use an import preset that imports animations to separate files."),
+			TTR("Warning: Editing imported animation").capitalize());
```

Comment:
I don't think we capitalize titles usually.

Reply:
Ok, took it out.